src/components/Member.tsx

```diff
@@ -0,0 +1,25 @@
+type Props = {
+  image: string;
+  name: string;
+  role: string;
+};
+
+const Member = ({ image, name, role }: Props) => {
+  return (
+    <>
+      <div className="flex flex-row h-16 gap-6 justify-center items-center">
```

Comment:
- don't give a height to the element it self the image already sets the height
- flex-row is unnecessary, that's the default
- gap-6 is not 20px like in the design